docs/guides/vault/create-a-vault.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_label: 'Create a Vault'
+sidebar_position: 1
+---
+
+# Create a Vault
+
+This guide will teach you how to deploy an **[LSP9Vault](../../standards/smart-contracts/lsp9-vault.md)** contract. This contract can be used to **hold assets** such as tokens and NFTs. Also can be used with a [UniversalProfile](../../standards/universal-profile/introduction.md) and a [KeyManager](../../standards/universal-profile/lsp6-key-manager.md) to **restrict some addresses** (protocols, friends, etc..) to execute and setData on it, instead of setting or executing directly on the profile.
+
+![Guide - How to create an LSP9Vault](../../../static/img/guides/LSP9VaultGuide.jpeg)
+
+## Deploy an LSP9Vault contract
+
+Check **[previous guides](../universal-profile/transfer-lyx.md#setup)** to learn how to connect to LUKSO's L16 network and load your EOA.
```

Comment:
```suggestion
Check **[previous guides](../universal-profile/create-profile#step-1---create-an-eoa)** to learn how to connect to LUKSO's L16 network and load your EOA.
```

This part explains how to create an address - the Transfert LYX doc doesn't explain it.